Casks/dotnet-sdk-2.1.802.rb

```diff
@@ -0,0 +1,28 @@
+cask 'dotnet-sdk-2.1.802' do
```

Comment:
The style of this repo is to auto-update minor SDK patches. Otherwise, very soon, we will have a very very big list of casks to manage.

```suggestion
cask 'dotnet-sdk-2.1.800' do
```

If you agree to this, the file also need to be renamed to `dotnet-sdk-2.1.800.rb`

Reply:
Hi @isen-ng I agree and have renamed the file and the cask name while keeping the intended version.

Is this ok?